modules/custom/social_demo/src/Plugin/DemoContent/Book.php

```diff
@@ -143,49 +143,4 @@ protected function prepareImage($uuid) {
     return $value;
   }
 
-  /**
-   * The function for creating a book from the demo content.
-   *
-   * @param \Drupal\node\Entity\Node $entity
-   *    The related entity.
-   * @param $book
-   *    The book.
-   */
-  public function createBookLink(Node $entity, $book) {
```

Comment:
@frankgraave, I assume this function was no longer necessary after refactoring took place? Is that why it is removed?

Reply:
You are correct! It's no longer necessary since @jochemvn worked his magic on refactoring it in a cleaner way!